### DISCUSSION TITLE
Issue with motion detection

### DISCUSSION THREAD

**Discussion by stahelp** [roleoroleo/sonoff-hack#58] [Q&A]
Hi

maybe someone from the community can help me with the following.
I have currently 2 Sonoff GK-200MP2-B cameras running and have the following issues:

Camera 1: Motion detection is enabled (via the app, set to high) but no videos are stored on the SD card and visible in the Sonoff-hack web frontend (motion-events).
Camera 2: Motion detection is enabled (via the app, set to high) and videos are recorded on the SD card and visible in the Sonoff-hack web frontend (motion-events). Usually after 10-12 hours recording stops and I need to restart the camera.

The configuration in the app and in the web frontend is the same for both cameras.
Both cameras have the same FW 2425.019130 and have the the Sonoff-hack 0.1.0 installed.

I'm I right, that the Sonoff app (eWeLink) is the only place where you can enable motion detection?
Any help is appreciated.

**Comment by roleoroleo** [comment]
I'm adding a setting in the web page to enable/disable motion detection.
But it doesn't change the result, I think.
The option simply send to the cam the same message sent from the app.
